test/presenters/hotwire_combobox/component_test.rb

```diff
@@ -0,0 +1,51 @@
+require 'test_helper'
+
+class HotwireCombobox::ComponentTest < ApplicationViewTestCase
+  setup do
+    @view = ActionView::Base.new(ActionController::Base.view_paths, {}, ActionController::Base.new)
```

Comment:
Because we're inheriting from `ApplicationViewTestCase` the view object should be accessible by just calling `#view` anywhere in the test cases. No need to initialize here.

We use this in the `Listbox::Option` tests in the `#render` method.